src/page/todo/Todo.tsx

```diff
@@ -1,11 +1,66 @@
-import { Outlet } from 'react-router-dom';
+import { useEffect, useState, useRef } from 'react';
+import { Link } from 'react-router-dom';
+
+import * as styles from './Todo.css';
+
+import GoButton from '@/common/component/GoButton/GoButton';
+
+const TYPING_DURATION = 4000;
+const FULL_TEXT = '66일 간 달성할 목표를 입력하고\n만다르트를 시작해보세요!';
```

Comment:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

```suggestion
const FULL_TEXT = '66일 간 달성할 목표를 입력하고\n만다라트를 시작해보세요!';
```

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ